README.md

```diff
@@ -2,13 +2,9 @@
 These are the setup scripts required to fill data into the internal database.
 
 ## Main scripts
-### sample.py
-* What does it do?
+### voting_rights/voting_rights.py
+Parses our compilation of voting laws from voting_rights/voting_rights_2016.csv and outputs SQL file voting_rights/voting_rights.sql which can be executed to truncate and repopulate the voting_rights MySQL table.
 
-* Pre-requisites
-
-* Installation instructions
-
-* How to run it?
-
-* Troubleshooting tips
\ No newline at end of file
+**Usage (from db-scripts directory):**
+* pip3 install -r voting_rights/requirements.txt
```

Comment:
requirements.txt is empty and gives a warning when you run this command which might be confusing to some.